py3status/modules/wwan_status_nm.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
+"""
+Display wwan network operator, signal and netgen properties,
+based on ModemManager, NetworkManager and dbus.
+
+Configuration parameters:
+    cache_timeout: How often we refresh this module in seconds.
+        (default 5)
+    consider_3G_degraded: If set to True, only 4G-networks will be
+        considered 'good'; 3G connections are shown
+        as 'degraded', which is yellow by default. Mostly
+        useful if you want to keep track of where there
+        is a 4G connection.
+        (default False)
+    format_down: What to display when the modem is down.
+        (default 'WWAN: {status} - {operator} {netgen} ({signal}%)')
+    format_error: What to display when the modem is not plugged in or on error.
+        (default 'WWAN: {error}')
+        available placeholder {error}
+    format_up: What to display upon regular connection
+    network available placeholders are {ip}, {ipv4_address}, {ipv4_dns1}, {ipv4_dns2},
+        {ipv6_address}, {ipv6_dns1}, {ipv6_dns2}
```

Comment:
This is hard to read. See `uname`. We want to list available placeholders below with new `Format placeholders:`.